### PR TITLE
Adding Travis-CI config

### PR DESCRIPTION
**Pull request by sandroacoelho**

### CODE REVIEW

**Comment by sandroacoelho** [comment]
I need to move all maven artifacts before. Working on it.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+language: java
+jdk:
+  - oraclejdk8
+  - oraclejdk7
+  - openjdk7
+
+
+# Workaround for https://github.com/travis-ci/travis-ci/issues/5227
```
Comment (sandroacoelho) [inline .travis.yml]: See travis-ci/travis-ci#5227 for more information.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# Buffer overflow in Java_java_net_Inet4AddressImpl_getLocalHostName
+before_install:
+  - cat /etc/hosts # optionally check the content *before*
+  - sudo hostname "$(hostname | cut -c1-63)"
+  - sed -e "s/^\\(127\\.0\\.0\\.1.*\\)/\\1 $(hostname | cut -c1-63)/" /etc/hosts | sudo tee /etc/hosts
+  - cat /etc/hosts # optionally check the content *after*
```
Comment (sandroacoelho) [inline .travis.yml]: See travis-ci/travis-ci#5227 for more information.

File: pom.xml [hunk 3]
```diff
@@ -15,13 +15,14 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <maven.scala.plugin.version>3.1.0</maven.scala.plugin.version>
         <maven.surefire.plugin.version>2.12.4</maven.surefire.plugin.version>
-        <scala.compiler.version>2.9.2</scala.compiler.version>
-        <scala.test.version>1.8</scala.test.version>
-        <dbpedia.extraction.version>3.8</dbpedia.extraction.version>
+        <scala.compiler.version.revision>2.11.7</scala.compiler.version.revision>
+        <scala.compiler.version>2.11</scala.compiler.version>
+        <scala.test.version>2.2.6</scala.test.version>
+        <dbpedia.extraction.version>4.0</dbpedia.extraction.version>
```
Comment (sandroacoelho) [inline pom.xml]: Using latest stable DBpedia Extraction Framework available at Maven Central

File: pom.xml [hunk 4]
```diff
@@ -15,13 +15,14 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <maven.scala.plugin.version>3.1.0</maven.scala.plugin.version>
         <maven.surefire.plugin.version>2.12.4</maven.surefire.plugin.version>
-        <scala.compiler.version>2.9.2</scala.compiler.version>
-        <scala.test.version>1.8</scala.test.version>
-        <dbpedia.extraction.version>3.8</dbpedia.extraction.version>
+        <scala.compiler.version.revision>2.11.7</scala.compiler.version.revision>
```
Comment (sandroacoelho) [inline pom.xml]: Upgrading Scala Compiler

**Comment by sandroacoelho** [comment]
Hi @jimkont,

Could you please review and merge?

Thanks

**Comment by jimkont** [comment]
+1

**Pull request merged**